src/main/java/fi/hsl/common/metro/MetroStops.java

```diff
@@ -0,0 +1,71 @@
+package fi.hsl.common.metro;
+
+import com.typesafe.config.Config;
+import fi.hsl.common.config.ConfigParser;
+import fi.hsl.common.transitdata.PubtransFactory;
+import org.apache.pulsar.shade.com.google.common.collect.ArrayListMultimap;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Optional;
+
+public class MetroStops {
+    private static final HashMap<String, String> shortNameByStopNumber = new HashMap<>();
+    private static final ArrayListMultimap<String, String> stopNumbersByShortName = ArrayListMultimap.create();
+    private static final List<String> shortNames = new ArrayList<>();
+
+    static {
+        final Config stopsConfig = ConfigParser.createConfig("metro_stops.conf");
+        stopsConfig.getObjectList("metroStops")
+                .forEach(stopConfigObject -> {
+                    final Config stopConfig = stopConfigObject.toConfig();
+                    final String shortName = stopConfig.getString("shortName");
+                    final List<String> stopNumbers = stopConfig.getStringList("stopNumbers");
+                    shortNames.add(shortName);
+                    stopNumbers.forEach(stopNumber -> {
+                        shortNameByStopNumber.put(stopNumber, shortName);
+                        stopNumbersByShortName.put(shortName, stopNumber);
+                    });
+                });
+    }
+
+    private MetroStops() {}
+
+    public static Optional<String> getShortName(final String stopNumber) {
+        return Optional.ofNullable(shortNameByStopNumber.get(stopNumber));
+    }
+
+    public static List<String> getStopNumbers(final String shortName) {
+        return stopNumbersByShortName.get(shortName);
+    }
+
+    public static Optional<Integer> getJoreDirection(final String startStop, final String endStop) {
```

Comment:
This method is not needed because we get the direction from Redis.

Reply:
Moved to metro-ats-parser